packages/react-rest/src/services/react-hooks.ts

```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect } from "react"
+import { getList, subscribeToList } from "data-core"
+import type { Source, Record, QueryList } from "data-core"
+
+export const useList = (
+  dataSource: Source,
+  schema: string,
+  query: QueryList,
+): [Record[]] => {
+  const [data, setData] = useState<Record[]>([]) // @todo default to what's in the store
+
+  useEffect(() => {
+    getList(dataSource, schema, query).then(setData)
+  }, [dataSource, schema, query])
+
+  useEffect(() => {
+    const unsubscribe = subscribeToList(schema, (records: Record[]) =>
+      setData(records),
+    )
+
+    return () => unsubscribe()
```

Comment:
since the return is just calling the function that the other function returned, you can just `return subscribeToList(...`